app/models/operatingsystems/coreos.rb

```diff
@@ -28,7 +28,11 @@ def mediumpath(medium_provider)
   end
 
   def pxedir(medium_provider = nil)
-    '$arch-usr/$version'
+    if medium_provider&.os_major&.to_s == "0" && medium_provider&.os_minor&.to_s == "0"
```

Comment:
The minor version should already be optional, so perhaps this is sufficient?
```suggestion
    if !medium_provider&.os_major || medium_provider&.os_major&.to_s == "0"
```